tests/src/ExistingSite/DecoupledTest.php

```diff
@@ -133,7 +133,7 @@ public function testResourceTypes(?array $roles) {
 
     // The user resource type should be disabled, so we should not be able to
     // do anything with it.
-    $this->assertResourceType(FALSE, $this->createUser(), [
```

Comment:
@apathak18 I think we disable the user resource intentionally. I think we should not enable it. Just fix the WSOD error and we are good.

@vishalkhode1 - what do you suggest.